DiskSetup.sh

```diff
@@ -55,7 +55,13 @@ function CarveLVM() {
 
    # Create LVM objects
    LVCSTAT=0
-   pvcreate "${CHROOTDEV}${PARTPRE}2" || LogBrk 5 "PV creation failed. Aborting!"
+   # Let's only attempt this if we're a secondary EBS
```

Comment:
Looks like you got your new patch wrapped up in here also?

Reply:
Wasn't actual intention. Only meaning to catch this one up to the other, higher-priority branch.

Normally, I do a merge from master so there's no "surprises". Since it wasn't in master, yet, I mered from the other branch (not knowing when that other branch would get merged).

Reply:
Just makes it harder to review when there are patches in a PR that are not intended to be merged with that PR.

Reply:
Err... Need me to roll back and re-push?

Reply:
It doesn't seem mergeable as it is. And if it's not mergeable, it's not really reviewable to me. So, I'd say yes.

Reply:
Rolling back...